mycurls/01_paypal_curl.txt

```diff
@@ -0,0 +1,43 @@
+curl -v -X POST https://api-m.sandbox.paypal.com/v2/checkout/orders \
```

Comment:
It is unclear what purpose these files serve, they look like local development notes that should not be committed into the repository.